debian/changelog

```diff
@@ -1,3 +1,9 @@
+raspberrypi-firmware (9.20201102-4.19.95+revpi1_0flat1) stable; urgency=medium
```

Comment:
9.20201102-4.19.95+revpi1_0flat1 => 9.20201102-4.19.95+revpi1~0flat1

The underscore is only used for tags. As git doesn't allow ~ in tag names.